contents/docs/feature-flags/best-practices.mdx

```diff
@@ -3,7 +3,17 @@ title: Feature flag best practices
 sidebar: Docs
 showTitle: true
 ---
-## 1. Call your flag in as few places as possible
+## 1. Use a reverse proxy
+
+Ad blockers can interfere with your feature flags, causing them to be disabled and users having a bad experience, or seeing the wrong version of your app.
+
+To avoid this, deploy a reverse proxy, which allows you to send events to PostHog Cloud using your own domain.
+
+This means that requests are less likely to be intercepted by tracking blockers, and your feature flags will work as intended. You'll also capture more usage data.
```

Comment:
Since reverse proxy isn't perfect, I don't know if we can say "flags will work as intended". More likely, sure.
```suggestion
This means that requests are less likely to be intercepted by tracking blockers, and your feature flags are more likely to work as intended. You'll also capture more usage data.
```